www/_posts/2019/2019-12-18-pyladies-global-council-process-vote.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post.html
+title: "PyLadies Global Council Selection Process Voting Now Open!"
+tags: [PyLadies, Global, Organizing]
+author: Lorena Mesa
+author_link: https://twitter.com/loooorenanicole
+---
+
+
+PyLadies Global Council Selection Process Voting Now Open!
+====================================================================================
+
+The PyLadies Global Interim team has now opened voting for PyLadies Chapter organizers to select one of three proposals for how the inaugural PyLadies Global Council election will function. 
+
+All information about voting can be found on the [PyLadies Global Organizing repository on issue #50](https://github.com/pyladies/global-organizing/issues/50).
+The proposals have been translated by volunteer community members to various languages: Arabic, French, Spanish, Portuguese, and Russian. We'd like to thank Python community members for stepping up and for helping PyLadies in the translation effort including Debora Azevedo, Debora Azevedo, Tania Allard, and volunteers in the Python Software Founduation Translation Workgroup.
```

Comment:
@Mariatta do you know the names of the volunteers you interacted with?

Reply:
Abdur-Rahmaan Janahangeer, Fadma Adam, Elizaveta Shashkova, Maria Ronacher, Philippe Gagnon